tests/har.spec.ts

```diff
@@ -322,7 +322,7 @@ it('should have security details', async ({ contextFactory, httpsServer, browser
   expect(serverIPAddress).toMatch(/^127\.0\.0\.1|\[::1\]/);
   expect(port).toBe(httpsServer.PORT);
   if (browserName === 'webkit' && platform === 'win32')
-    expect(securityDetails).toEqual({subjectName: 'puppeteer-tests', validFrom: 1550084863});
+    expect(securityDetails).toEqual({subjectName: 'puppeteer-tests', validFrom: 1550084863, validTo: -1});
```

Comment:
Not sure what the best approach is here…should I just mark this test as fully failing on Win32? `filterEmpties` used to filter out the -1's we were getting from WK.